generatorreceiver/internal/topology/service_route.go

```diff
@@ -8,13 +8,12 @@ import (
 )
 
 type ServiceRoute struct {
-	Route                 string                 `json:"route" yaml:"route"`
-	DownstreamCalls       []Call                 `json:"downstreamCalls,omitempty" yaml:"downstreamCalls,omitempty"`
-	MaxLatencyMillis      int64                  `json:"maxLatencyMillis" yaml:"maxLatencyMillis"`
-	LatencyConfigs        LatencyConfigs         `json:"latencyConfigs" yaml:"latencyConfigs"`
-	TagSets               []TagSet               `json:"tagSets" yaml:"tagSets"`
-	ResourceAttributeSets []ResourceAttributeSet `json:"resourceAttrSets" yaml:"resourceAttrSets"`
```

Comment:
Was this supposed to be deleted?

Reply:
Ah, yes - the `ResourceAttributeSets` member of ServiceRoute` wasn't used anywhere in the code, so I thought it best to remove until we decide to implement it, if we do.

Reply:
Ohhh! I see, for a moment I thought this was on the `ServiceTier` struct. Since this is not used, I agree with removing it!